docs/source/en/model_doc/vitpose.md

```diff
@@ -10,277 +10,107 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
-                    cv2.addWeighted(image, transparency, image, 1 - transparency, 0, dst=image)
-                else:
-                    cv2.line(image, (x1, y1), (x2, y2), color, thickness=thickness)
-
-
-# Note: keypoint_edges and color palette are dataset-specific
-keypoint_edges = model.config.edges
-
-palette = np.array(
-    [
-        [255, 128, 0],
-        [255, 153, 51],
-        [255, 178, 102],
-        [230, 230, 0],
-        [255, 153, 255],
-        [153, 204, 255],
-        [255, 102, 255],
-        [255, 51, 255],
-        [102, 178, 255],
-        [51, 153, 255],
-        [255, 153, 153],
-        [255, 102, 102],
-        [255, 51, 51],
-        [153, 255, 153],
-        [102, 255, 102],
-        [51, 255, 51],
-        [0, 255, 0],
-        [0, 0, 255],
-        [255, 0, 0],
-        [255, 255, 255],
-    ]
-)
-
-link_colors = palette[[0, 0, 0, 0, 7, 7, 7, 9, 9, 9, 9, 9, 16, 16, 16, 16, 16, 16, 16]]
-keypoint_colors = palette[[16, 16, 16, 16, 16, 9, 9, 9, 9, 9, 9, 0, 0, 0, 0, 0, 0]]
+from optimum.intel import IncQuantizer
 
-numpy_image = np.array(image)
+quantizer = IncQuantizer.from_pretrained("usyd-community/vitpose-base-simple")
+quantizer.quantize(save_directory="./vitpose-quantized")
+```
 
-for pose_result in image_pose_result:
-    scores = np.array(pose_result["scores"])
-    keypoints = np.array(pose_result["keypoints"])
+---
 
-    # draw each point on image
-    draw_points(numpy_image, keypoints, scores, keypoint_colors, keypoint_score_threshold=0.3, radius=4, show_keypoint_weight=False)
+## Attention Visualization
 
-    # draw links
-    draw_links(numpy_image, keypoints, scores, keypoint_edges, link_colors, keypoint_score_threshold=0.3, thickness=1, show_keypoint_weight=False)
+```python
+from transformers.utils.attention_mask_visualizer import AttentionMaskVisualizer
 
-pose_image = Image.fromarray(numpy_image)
-pose_image
+visualizer = AttentionMaskVisualizer(model)
+visualizer.visualize(inputs["attention_mask"])
 ```
-<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/transformers/model_doc/vitpose-coco.jpg" alt="drawing" width="600"/>
 
-## Resources
+---
+
+## Notes
 
-A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with ViTPose. If you're interested in submitting a resource to be included here, please feel free to open a Pull Request and we'll review it! The resource should ideally demonstrate something new instead of duplicating an existing resource.
+- ViTPose is a top-down pose estimator: it requires a prior object detector to crop individuals before keypoint prediction.
+- ViTPose++ supports passing a `dataset_index` to activate different MoE expert heads.
+- Use `AutoProcessor` to handle bounding box preprocessing automatically.
+- Ideal for use cases involving MS COCO, WholeBody, MPII, AP-10K, and APT-36K keypoint datasets.
```

Comment:
Use the `<hfoptions>...</hfoptions>` tags under this to add code snippets for supervision and OpenCV.

```suggestion
- Use the [supervision](https://github.com/roboflow/supervision) or [OpenCV](https://opencv.org/) libraries to visualize the keypoints.
```